### DISCUSSION TITLE
It’s time for us to talk about creating AI-free spaces

### DISCUSSION THREAD

**Discussion by Aaron25564448** [1101-GS4719-C/Python-Community-GS4719-C#1462] [General]
https://theconversation.com/its-time-for-us-to-talk-about-creating-ai-free-spaces-202176
在這個AI到處參一腳、到處搶工作的社會現況下，針對AI的氾濫，有人提出了「AI避難所」，也就是限制AI活動的領域。正是因為目前AI的發展若是如此持續下去，極有可能危害到人類的隱私及認知能力。這個想法在我看來可以說是非常新穎，在某些領域確實是有要限制的必要。像是AI繪圖近幾年就引起不少爭議，也正是需要限制AI使用的領域之一。AI繪圖的機制大多是基於現有畫作進行學習、融合，在是否侵害原作者著作權依舊有待商議；而在這方面，有不少平台也要求標示畫作是否為AI創作，這點雖然是自由心證，但確實令人、包括我，放心不少。希望AI與人類之間的互動能盡快找到一個平衡點。

**Comment by aa103967622** [comment]
據我所知，目前國際上針對ai創造出來的作品是否受著作權保護，有著一定的判斷標準，以「創作」的過程中，是否有人為的「創作意圖」及「實際對創作的參與」，可以分為以人工智慧為工具的創作以及人工智慧獨立創作。比如，在人賦予ai畫圖的關鍵字時，應該包含自己的創作理念，或者是以ai畫的圖為基礎，後續進行修改，這樣就可以保證你利用ai為工具創作出來的作品之著作權。
